### PR TITLE
gwenhywfar 5.2.0, aqbanking 6.1.2

### PR DESCRIPTION
**Pull request by chenrui333**
bump the version.

previous upgrade, #44388

### CODE REVIEW

**Comment by Bo98** [comment]
The tests will probably fail. The format changed and I don't know enough about German banking to figure it out.

**Comment by chenrui333** [comment]
yeah, it indeed failed. Let me try locally for this PR later.

**Comment by chenrui333** [comment]
test failure for gwenhywfar
build failure for aqbanking

**Comment by chenrui333** [comment]
~test failure for gwenhywfar~ checked locally

**Comment by Bo98** [comment]
Oh I managed to get it to build - just not pass the test.

You want to remove `--with-gwen-dir`. It should work automatically without it.

**Comment by chenrui333** [comment]
Right now I am struggling with the `aqbanking-cli` test block, not sure how to fix that.

Just pushed what i have.

File: Formula/aqbanking.rb [hunk 1]
```diff
@@ -29,13 +29,16 @@ class Aqbanking < Formula
   depends_on "libxslt"
 
   def install
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["gwenhywfar"].lib/"pkgconfig"
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["libxmlsec1"].lib
```
Comment (Bo98) [inline Formula/aqbanking.rb]: Bit odd you needed this. I don't recall adding it.

Comment (chenrui333) [reply Formula/aqbanking.rb]:
```
==> ./configure --prefix=/usr/local/Cellar/aqbanking/6.1.2 -I/usr/local/opt/libxmlsec1/include/xmlsec1 --enable-cli
Last 15 lines from /Users/ruichen/Library/Logs/Homebrew/aqbanking/01.configure:
2020-03-11 22:57:47 -0400

./configure
--disable-debug
--disable-dependency-tracking
--prefix=/usr/local/Cellar/aqbanking/6.1.2
-I/usr/local/opt/libxmlsec1/include/xmlsec1
--enable-cli

configure: error: unrecognized option: `-I/usr/local/opt/libxmlsec1/include/xmlsec1'
Try `./configure --help' for more information
```

Comment (chenrui333) [reply Formula/aqbanking.rb]: that is the error I got after adding L32

Comment (Bo98) [reply Formula/aqbanking.rb]: My diff before I abandoned it was simply:

```diff
diff --git a/Formula/aqbanking.rb b/Formula/aqbanking.rb
index d96961cdc8..3fd808965b 100644
--- a/Formula/aqbanking.rb
+++ b/Formula/aqbanking.rb
@@ -1,8 +1,8 @@
 class Aqbanking < Formula
   desc "Generic online banking interface"
   homepage "https://www.aquamaniac.de/sites/aqbanking/"
-  url "https://www.aquamaniac.de/rdm/attachments/download/107/aqbanking-5.8.2.tar.gz"
-  sha256 "93ca523fe175e72042db75f8c3fc6255ab058cf82caf52796e15f030809fb15e"
+  url "https://www.aquamaniac.de/rdm/attachments/download/269/aqbanking-6.1.2.tar.gz"
+  sha256 "0b54c70a8ab1bd5537d8b26d05a9e44db0600b64d8cbb63c34018dee11f67190"
 
   bottle do
     sha256 "95a5722e1489e556d98a230d56ddf577f31916e9d7d92624122d3b9e47d9db17" => :catalina
@@ -29,19 +29,17 @@ class Aqbanking < Formula
   depends_on "libxslt"
 
   def install
-    ENV.deparallelize
     system "autoreconf", "-fiv" if build.head?
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
-                          "--enable-cli",
-                          "--with-gwen-dir=#{HOMEBREW_PREFIX}"
-    system "make", "check"
+                          "--enable-cli"
     system "make", "install"
   end
 
   test do
     ENV["TZ"] = "UTC"
+    ENV["AQBANKING_HOME"] = testpath/".aqbanking"
     context = "balance.ctx"
     (testpath/context).write <<~EOS
       accountInfoList {
```

Comment (chenrui333) [reply Formula/aqbanking.rb]: `+    ENV["AQBANKING_HOME"] = testpath/".aqbanking"`
does not work for me though

installation process stays the same

Comment (chenrui333) [reply Formula/aqbanking.rb]:
```
==> /usr/local/Cellar/aqbanking/6.1.2/bin/aqbanking-cli listbal -c balance.ctx
3:2020/03/14 02-47-46:(null)(51834):./banking_update.c:  610: No AqBanking config folder found at [/Users/ruichen/.aqbanking/settings6/users] (-1)
3:2020/03/14 02-47-46:(null)(51834):./banking_update.c:  610: No AqBanking config folder found at [/Users/ruichen/.aqbanking/settings/users] (-1)
3:2020/03/14 02-47-46:(null)(51834):./banking_update.c:  411: There is no old settings folder, need initial setup
Error: aqbanking: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: </Account\ 110000000\ 000123456789\ STRIPE\ TEST\ BANK\ 03\.01\.2014\ 12:00\ 1324\.36\ USD/> was expected to be =~
<"">.
```

Comment (chenrui333) [reply Formula/aqbanking.rb]: Could not find anything online related to the settings6

Comment (Bo98) [reply Formula/aqbanking.rb]: Oh I never said I passed the test, that line just got it one stage further. It gave a different test error without it.

The problem is the format has changed in that big file that gets written in the tests. Also `listbal` I think behaves slightly differently.

**Pull request closed without merging**